CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Change Log
 
+## Version 0.2.0
+_2018-04-08_
```

Comment:
Minor comment:
I think this date should be 2019-04-08?

Reply:
Ahhhh, my bad, copy&paste mistake.

Reply:
Fixed on master, but too late for the published pod. Thanks @nevil !